pandas/sparse/array.py

```diff
@@ -620,19 +620,28 @@ def sum(self, axis=0, *args, **kwargs):
 
     def cumsum(self, axis=0, *args, **kwargs):
         """
-        Cumulative sum of values. Preserves locations of NaN values
+        Cumulative sum of non-NA/null values.
+
+        When performing the cumulative summation, any non-NA/null values will
+        be skipped. The resulting SparseArray will preserve the locations of
+        NaN values, but the fill value will be `np.nan` regardless.
+
+        Parameters
+        ----------
+        axis : int
```

Comment:
@jreback : The parameter was in the signature but undocumented.  I believe it is for compat since `SparseDataFrame` actually uses this argument.